src/_cffi_src/openssl/src/osrandom_engine.h

```diff
@@ -0,0 +1,71 @@
+#ifdef _WIN32
+  #include <Wincrypt.h>
+#else
+  #include <fcntl.h>
+  #include <unistd.h>
+   /* for defined(BSD) */
+  #include <sys/param.h>
+
+  #ifdef BSD
```

Comment:
Does FreeBSD define this? I think they typically use `__FreeBSD__`. What about macOS (which apparently has `SYS_getentropy` as of 10.12)?